### ISSUE TITLE
On Debian 7.4 it doesn't work properly

### ISSUE THREAD

**Issue by bogdanr** [saltstack-formulas/nginx-formula#14]
As you can see, adding the service fails on Debian 7.4:

```
root@2boffice:~# salt 'sites-ruby' state.sls nginx
sites-ruby:
----------
          ID: /usr/share/nginx
    Function: file.directory
      Result: True
     Comment: Directory /usr/share/nginx is in the correct state
     Changes:   
----------
          ID: /etc/nginx/conf.d/default.conf
    Function: file.absent
      Result: True
     Comment: File /etc/nginx/conf.d/default.conf is not present
     Changes:   
----------
          ID: /etc/nginx/conf.d/example_ssl.conf
    Function: file.absent
      Result: True
     Comment: File /etc/nginx/conf.d/example_ssl.conf is not present
     Changes:   
----------
          ID: /var/log/nginx/access.log
    Function: file.absent
      Result: True
     Comment: File /var/log/nginx/access.log is not present
     Changes:   
----------
          ID: nginx-logger-access
    Function: file.managed
        Name: /etc/init/nginx-logger-access.conf
      Result: True
     Comment: File /etc/init/nginx-logger-access.conf is in the correct state
     Changes:   
----------
          ID: nginx-logger-access
    Function: service.running
      Result: False
     Comment: The named service nginx-logger-access is not available
     Changes:   
----------
          ID: /var/log/nginx/error.log
    Function: file.absent
      Result: True
     Comment: File /var/log/nginx/error.log is not present
     Changes:   
----------
          ID: nginx-logger-error
    Function: file.managed
        Name: /etc/init/nginx-logger-error.conf
      Result: True
     Comment: File /etc/init/nginx-logger-error.conf is in the correct state
     Changes:   
----------
          ID: nginx-logger-error
    Function: service.running
      Result: False
     Comment: The named service nginx-logger-error is not available
     Changes:   
----------
          ID: /etc/logrotate.d/nginx
    Function: file.absent
      Result: True
     Comment: File /etc/logrotate.d/nginx is not present
     Changes:   
----------
          ID: /etc/nginx
    Function: file.directory
      Result: True
     Comment: Directory /etc/nginx is in the correct state
     Changes:   
----------
          ID: /etc/nginx/nginx.conf
    Function: file.managed
      Result: True
     Comment: File /etc/nginx/nginx.conf is in the correct state
     Changes:   
----------
          ID: /etc/nginx/sites-enabled
    Function: file.directory
      Result: True
     Comment: Directory /etc/nginx/sites-enabled is in the correct state
     Changes:   
----------
          ID: /etc/nginx/sites-available
    Function: file.directory
      Result: True
     Comment: Directory /etc/nginx/sites-available is in the correct state
     Changes:   
----------
          ID: htpasswd
    Function: pkg.installed
        Name: apache2-utils
      Result: True
     Comment: Package apache2-utils is already installed
     Changes:   
----------
          ID: nginx-old-init
    Function: file.rename
        Name: /usr/share/nginx/init.d
      Result: True
     Comment: Source file "/etc/init.d/nginx" has already been moved out of place
     Changes:   
----------
          ID: nginx-old-init
    Function: module.wait
        Name: cmd.run
      Result: True
     Comment: 
     Changes:   
----------
          ID: nginx-old-init
    Function: cmd.wait
        Name: dpkg-divert --divert /usr/share/nginx/init.d --add /etc/init.d/nginx
      Result: True
     Comment: 
     Changes:   
----------
          ID: nginx-old-init-disable
    Function: cmd.wait
        Name: update-rc.d -f nginx remove
      Result: True
     Comment: 
     Changes:   
----------
          ID: nginx
    Function: pkg.installed
      Result: True
     Comment: Package nginx is already installed
     Changes:   
----------
          ID: nginx
    Function: file.managed
        Name: /etc/init/nginx.conf
      Result: True
     Comment: File /etc/init/nginx.conf is in the correct state
     Changes:   
----------
          ID: nginx
    Function: service.running
      Result: False
     Comment: One or more requisite failed
     Changes:   

Summary
-------------
Succeeded: 19
Failed:     3
-------------
Total:     22
```

**Comment by ninapavlich** [comment]
I'm seeing this problem on Ubuntu 17.04

**Comment by noelmcloughlin** [comment]
Resolved in #15 via `use_upstart: True`
